docs/configuration.md

```diff
@@ -167,6 +167,8 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `E2E_RELAY_HOST` | ""      | Remote relay url to point to.                                |
 | `DEV_MODE`       | "false" | Flag if relay should operate in developer optimization mode. |
 | `TEST_WS_SERVER` | "false" | Flag config for enable or disable the WS server tests.       |
+| `LOWER_GAS_PRICE_DEVIATION` | 0.80 | Lower value to use when comparing gas prices in the rpc-batch1.spec.ts |
```

Comment:
Also let's use a `TEST_` prefix to not this is test config not product config

Reply:
Done.